### PR TITLE
Added Confidential field to Epic struct

### PR DESCRIPTION
**Pull request by MakingPRs**
The confidentiality is available as a field within Epics https://docs.gitlab.com/ee/api/epics.html

### CODE REVIEW

**Comment by MakingPRs** [comment]
from https://github.com/xanzy/go-gitlab/issues/1652

File: epics.go [hunk 1]
```diff
@@ -69,6 +69,7 @@ type Epic struct {
 	Downvotes               int         `json:"downvotes"`
 	UserNotesCount          int         `json:"user_notes_count"`
 	URL                     string      `json:"url"`
+	Confidential		bool	    `json:"confidential"`
```
Comment (svanharmelen) [inline epics.go:72]: Please place the field between `State` and `WebURL` following the ordering in the docs...

Comment (MakingPRs) [reply epics.go]: Yes, done!

**Review by svanharmelen — APPROVED** [review]
Thanks 👍🏻

**Pull request merged**